docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+[versions of pip, any other tools, etc. go here]
+
+If you haven't done so already, install the gRPC runtime and gRPC tools.
+
+### Install gRPC
+
+Install gRPC:
+
+```sh
+  $ pip install grpcio
+```
+
+Or, to install it system wide:
+
+```sh
+  $ sudo pip install grpcio
+```
+
+If you're on Windows, make sure you installed the `pip.exe` component when you
+installed Python. Invoke as above but with `pip.exe` instead of `pip` (you may
+also need to invoke from a `cmd.exe` ran as administrator):
+
+```sh
+  $ pip.exe install grpcio
+```
+
+### Install gRPC tools
+
+Python's gRPC tools include the protocol buffer compiler `protoc` and the special plugin for generating server and client code from `.proto` gRPC service definitions. For the first part of our quickstart example, we've already generated the server and client stubs from [helloworld.proto](https://github.com/grpc/grpc/tree/{{ site.data.config.grpc_release_branch }}/examples/protos/helloworld.proto), but you'll need the tools for the rest of our quickstart, as well as later tutorials and your own projects.
+
+To install gRPC tools, run:
+
+```sh
+pip install grpcio-tools
+
+```
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository:
+
+```sh
+  $ # Clone the repository to get the example code:
+  $ git clone https://github.com/grpc/grpc
+  $ # Navigate to the "hello, world" Python example:
+  $ cd grpc/examples/python/helloworld
+  ```
+
+
+## Run a gRPC application
+
+From the `examples/python/helloworld` directory:
+
+1. Run the server
+
+   ```sh
+   $ python2.7 greeter_server.py &
+   ```
+
+2. In another terminal, run the client
+
+   ```sh
+   $ python2.7 greeter_client.py
+   ```
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the server for the client to call. Our gRPC service is defined using protocol buffers: you can find out lots more about how this works in [What is gRPC?]() and [gRPC Basics: Python][]. For now all you need to know is that both the server and the client "stub" have a `SayHello` RPC method that takes a `HelloRequest` parameter from the client and returns a `HelloResponse` from the server, and that this method is defined like this:
```

Comment:
If possible please write this text on several lines in the source markdown file; I think it will still render as an ordinary paragraph in the reader-facing form.